services/src/unit-test/java/uk/gov/moj/sdt/services/mbeans/SdtManagementMBeanTest.java

```diff
@@ -114,207 +144,178 @@ public class SdtManagementMBeanTest extends AbstractSdtUnitTestBase {
      */
     private ISdtManagementMBean sdtManagementMBean;
 
-    /**
-     * Mock Individual Request Dao to perform operations on the individual request object.
-     */
-    private IIndividualRequestDao mockIndividualRequestDao;
-
-    /**
-     * This mock messaging utility reference for testing.
-     */
-    private IMessagingUtility mockMessagingUtility;
-
-    /**
-     * This mock variable holding the target application submission service.
-     */
-    private ITargetApplicationSubmissionService mockTargetAppSubmissionService;
-
     /**
      * Method to do any pre-test set-up.
      */
-    @Before
+    @BeforeEach
+    @Override
     public void setUp() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
This should be able to be removed as the ExtendWith(MockitoExtension.class) annotation initialises the annotated mocks in the class.

Reply:
Confirmed openMocks() line removed.